contracts/Auctioneer.sol

```diff
@@ -30,6 +30,7 @@ contract Auctioneer is CloneFactory {
     // which will be used as a master contract for cloning.
     address public masterAuction;
     mapping(address => bool) public openAuctions;
+    uint256 public openAuctionsCount;
```

Comment:
WDYT about adding `hasOpenAuctions` function?

Reply:
Done as part of https://github.com/keep-network/coverage-pools/pull/88/commits/3f36835113dac64bdd05101fd10fd69b0c143f3a